src/mudblazor/MudBlazor.Template/MudBlazor.Template.csproj

```diff
@@ -7,22 +7,21 @@
     <UserSecretsId Condition="'$(IndividualLocalAuth)' == 'True'">aspnet-MudBlazor.Template-53bc9b9d-9d6a-45d4-8429-2a2761773502</UserSecretsId>
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">MudBlazor.Template</RootNamespace>
-    <AssemblyName Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">$(AssemblyName.Replace(' ', '_'))</AssemblyName>
+    <AssemblyName Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">`$(AssemblyName.Replace(' ', '_'))</AssemblyName>
   </PropertyGroup>
 
   <!--#if (IndividualLocalAuth && !UseLocalDB) -->
   <ItemGroup>
-    <None Update="app.db" CopyToOutputDirectory="PreserveNewest" ExcludeFromSingleFile="true" />
+    <None Update="Data\app.db" CopyToOutputDirectory="PreserveNewest" ExcludeFromSingleFile="true" />
   </ItemGroup>
 
   <!--#endif -->
-  
+
   <!--#if (!UseWebAssembly && !IndividualLocalAuth) -->
   <ItemGroup>
-    <PackageReference Include="MudBlazor" Version="6.*" />
+    <PackageReference Include="MudBlazor" Version="7.0.0-rc.2" />
   </ItemGroup>
   <!--#endif -->
-  
   <!--#if (UseWebAssembly || IndividualLocalAuth) -->
   <ItemGroup>
     <ProjectReference Include="..\MudBlazor.Template.Client\MudBlazor.Template.Client.csproj" Condition="'$(UseWebAssembly)' == 'True'" />
```

Comment:
I know this isn't a change and maybe based on Microsoft's templates, but for me, it doesn't look right to have a  `ProjectReference` and `PackageReference` in the same `ItemGroup`.

Reply:
That's how MS did it, and I guess I would prefer to leave it as is.